app/helpers/public_document_routes_helper.rb

```diff
@@ -112,6 +112,42 @@ def take_part_page_url(object, options = {})
     Plek.new.website_root + take_part_page_path(object, options)
   end
 
+  def topical_event_path(object, options = {})
+    slug = case object
+           when String
+             object
+           when TopicalEvent
+             object.slug
+           else
+             raise ArgumentError, "Must provide a slug or TopicalEvent"
+           end
+
+    append_url_options("/government/topical-events/#{slug}", options)
+  end
+
+  def topical_event_url(object, options = {})
+    Plek.new.website_root + topical_event_path(object, options)
+  end
+
+  def topical_event_about_pages_path(object, options = {})
+    slug = case object
+           when String
+             object
+           when TopicalEvent
+             object.slug
+           when TopicalEventAboutPage
+             object.topical_event.slug
+           else
+             raise ArgumentError, "Must provide a slug, TopicalEvent or TopicalEventAboutPage"
+           end
+
+    append_url_options("/government/topical-events/#{slug}/about", options)
+  end
+
+  def topical_event_about_pages_url(object, options = {})
+    Plek.new.website_root + topical_event_about_pages_path(object, options)
```

Comment:
Do we need this method? As far as I can tell it's only used in tests. Or do we want to make sure all content types we introduce helpers for have both 'url' and 'path' helpers for consistency with how it is when we have the rails helpers?

Reply:
I'll try removing this method and seeing if any of the existing tests fail.  I think I only added it for consistency, but will definitely remove it if it's not being used anywhere.

Reply:
All the tests passed, so I've removed the `topical_event_about_pages_url` method and its associated tests.  Also rebased against main as this branch was quite a bit out of date.